app/domain/operations/authentication/logout_saml_user.rb

```diff
@@ -9,6 +9,7 @@ class LogoutSamlUser
       include Dry::Monads[:do, :result, :try]
 
       def call(session)
+        return Success(:ok) unless EnrollRegistry.feature_enabled?(:saml_single_logout)
```

Comment:
this will efectively disable to try to logout on any version unless the setting is enabled, that means if we are using a lower version or another sso without the configuration for logout like keycloack will not try to logout.

it looks like a specific implementation for keycloack 18. 

The obvious example is that DC will not try to logout, I understand that DC is not on this repo anymore but looking at the future if we got another client with SSO we will probably have to rewrite this.

Reply:
There is no logout feature that is actually executed in the DC scenario - only a redirect.  There was never a logout - only a redirect.  The redirect remains intact.